components/base/kinematicbase/kinematics.go

```diff
@@ -23,7 +23,6 @@ type KinematicBase interface {
 	motion.Localizer
 	framesystem.InputEnabled
 
-	Kinematics() referenceframe.Frame
```

Comment:
Do we not need to replace this at this time?

Reply:
it's in `framesystem.InputEnabled` it is replaced